### PR TITLE
feat(parsers): add a parser to parse boolean with a default value true

### PR DESCRIPTION
**Pull request by aurrelhebert**

### CODE REVIEW

File: src/parsers.js [hunk 1]
```diff
@@ -191,6 +191,20 @@ function durationInSeconds (durationStr = '') {
   }
 }
 
+/**
+ * Parse a boolean input which is expect a default value as true
+ * An empty value returns true
+ * @returns {bool} true or false
```
Comment (pdesoyres-cc) [inline src/parsers.js:197]: `{bool}` type doesn't exist. you should use `{boolean}` instead.

**Review by pdesoyres-cc — CHANGES_REQUESTED** [review]
LGTM with just a little comment

**Comment by miton18** [comment]
Do we need a bool parser ?

**Comment by aurrelhebert** [comment]
Closing as related to the `watch` param we decided not to support

**Pull request closed without merging**